deck/views.py

```diff
@@ -204,6 +204,17 @@ def dispatch(self, *args, **kwargs):
         return super(CreateProposal, self).dispatch(*args, **kwargs)
 
 
+class MyProposals(BaseProposalView, ListView):
+    template_name = 'proposal/my_proposals.html'
+
+    def get_queryset(self):
+        return Proposal.objects.filter(author=self.request.user)
```

Comment:
`return self.request.user.proposals` or `Proposal.objects.filter(author_id=self.request.user.id)` for being more performatic, i do prefer the first version, it is more readable.